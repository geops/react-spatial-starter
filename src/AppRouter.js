import React from 'react';
import { BrowserRouter as Router, Route } from 'react-router-dom';
import App from './App';

const AppRouter = () => (
  <Router>
    <>
      <Route
        exact
        path="/"
        component={({ history, match }) => (
          <App history={history} initialState={match.params} />
        )}
      />
    </>
  </Router>
);

export default AppRouter;
