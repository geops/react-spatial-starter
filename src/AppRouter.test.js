import 'jest-canvas-mock';
import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import AppRouter from './AppRouter';

configure({ adapter: new Adapter() });

describe('AppRouter', () => {
  test('should match snapshot', () => {
    const component = renderer.create(<AppRouter />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
