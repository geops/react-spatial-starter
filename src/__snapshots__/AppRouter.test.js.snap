// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRouter should match snapshot 1`] = `
<div
  className="app"
>
  <div
    className="tm-root"
  >
    <h1>
      My react-spatial app centered on 922747.8054581558,5911639.7675754195
    </h1>
    <div
      className="tm-map"
      role="menu"
      tabIndex={0}
    />
  </div>
</div>
`;
